gustaf/helpers/options.py

```diff
@@ -0,0 +1,428 @@
+"""gustaf/gustaf/helpers/options.py
```

Comment:
Seems to be working as expected (well except for the vedo errors). I have not played around with the new options to get a feels for them so I can not say how robust the new system is.

Reply:
will add pr comments